app/models/ReleasedMovieWithID.scala

```diff
@@ -0,0 +1,18 @@
+package models
+
+import java.time.LocalDateTime
+
+import akka.http.javadsl.model.DateTime
+import play.api.libs.json.OFormat
+import reactivemongo.bson.BSONObjectID
+
+case class ReleasedMovieWithID(
+                      var _id: BSONObjectID,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name '_id' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5277123)